firelyserver/releasenotes/releasenotes.rst

```diff
@@ -13,10 +13,91 @@ Release notes Firely Server
 .. _upgrade_vonk:
 
 Upgrading Firely Server
---------------
+-----------------------
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Firely Server.
 
+.. _vonk_releasenotes_400:
+
+Release 4.0.0
+-------------
+
+This major version introduces a new name: **Firely Server** instead of Vonk. Other than that, this release contains some significant code changes, which could impact you if you run Firely Server with your own plugins.
+
+Features
+^^^^^^^^
+
+#. Name change Vonk -> Firely Server:
+   #. The main entry point dll (formerly: ``Vonk.Server.dll``) and executable (formerly: ``Vonk.Server.exe``) names have been changed to ``Firely.Server.dll`` and ``Firely.Server.exe`` respectively.
+   #. The name was changed in the CapabilityStatement.name.
+   #. The name of the download zip (from Simplifier) has changed from `vonk_distribution.zip` to `firely-server-latest.zip`. Likewise the versioned zip files have changed as well.
+#. We have implemented FHIR Bulk Data Access (``$export``) to allow for fast, asynchronous ndjson data exports. The :ref:`Bulk Data Export documentation<feature_bulkdataexport>` can help you to get started.
+#. Firely Server now uses Firely .NET SDK 2.0.2 (formerly: FHIR .NET API)
+
+   .. attention::
+   
+      If you are running Firely Server with your own self-made plugins, you will likely encounter package versioning problems and need to upgrade your NuGet Firely Server package references (package names starting with ``Vonk.``) to version 4.0.0. You also need to upgrade any Firely .NET SDK package references (package names starting with ``Hl7.Fhir.``) to version 2.0.2. The `Firely .NET SDK release notes <https://github.com/FirelyTeam/firely-net-sdk/releases>`_ and `Breaking changes in Firely SDK 2.0 <https://github.com/FirelyTeam/firely-net-sdk/wiki/Breaking-changes-in-2.0>`_ can give you an idea of the changes you may encounter in the SDK.
```

Comment:
This makes it sound like it's a bad thing to do, which we don't mean. I recommend:

```suggestion
      In case you've tailored your Firely Server with custom plugins, you will also need to upgrade your NuGet Firely Server package references (package names starting with ``Vonk.``) to version ``4.0.0``. You also need to upgrade any Firely .NET SDK package references (package names starting with ``Hl7.Fhir.``) to version 2.0.2. The `Firely .NET SDK release notes <https://github.com/FirelyTeam/firely-net-sdk/releases>`_ and `Breaking changes in Firely SDK 2.0 <https://github.com/FirelyTeam/firely-net-sdk/wiki/Breaking-changes-in-2.0>`_ can give you an idea of the changes you may encounter in the SDK.
```